beacon_node/src/config.rs

```diff
@@ -36,20 +38,12 @@ pub fn get_config<E: EthSpec>(
     // If necessary, remove any existing database and configuration
     if client_config.data_dir.exists() && cli_args.is_present("purge-db") {
         // Remove the chain_db.
-        let chain_db = client_config.get_db_path().ok_or("Failed to get db_path")?;
-        if chain_db.exists() {
-            fs::remove_dir_all(chain_db)
-                .map_err(|err| format!("Failed to remove chain_db: {}", err))?;
-        }
+        fs::remove_dir_all(client_config.get_db_path())
```

Comment:
Should we continue to check if the path exists before removing it?

Reply:
Yeah good catch, Mac added this in #2747